src/components/Book.jsx

```diff
@@ -37,35 +38,45 @@ const Book = () => {
   }
 
   return (
-    <div>
+    <div className="books-container">
       {/* Render the book data */}
       {booksArray.map((book) => (
         <div className="book" key={book.id}>
-          <article>
+          <article className="container-1">
             <div className="book-details">
-              <p>{book.category}</p>
-              <p>{book.title}</p>
-              <p>{book.author}</p>
+              <p className="category">{book.category}</p>
+              <p className="title">{book.title}</p>
+              <p className="author">{book.author}</p>
             </div>
             <div className="interactions">
-              <button type="submit">Comments</button>
-              <button type="submit" onClick={() => handleDeleteBook(book.id)}>Remove</button>
-              <button type="submit">Edit</button>
+              <button type="submit" className="btn">Comments</button>
+              <span className="line" />
+              <button type="submit" className="btn" onClick={() => handleDeleteBook(book.id)}>Remove</button>
+              <span className="line" />
+              <button type="submit" className="btn">Edit</button>
             </div>
           </article>
-          <article>
-            <div>circle</div>
+          <article className="container-2">
+            <div className="progress-img-container">
+              <ProgressBar />
+            </div>
             <div className="progress-text">
-              <p>Book.progress</p>
-              <p>Completed</p>
+              <p className="percentage">
+                {Math.floor(Math.random() * 101)}
+                %
+              </p>
+              <p className="complete">Completed</p>
             </div>
           </article>
           <article>
+            <div className="vertical-bar" />
+          </article>
+          <article className="container-4">
             <div>
-              <p>CURRENT CHAPTER</p>
-              <p>Book.chapter</p>
+              <p className="current-chapter">CURRENT CHAPTER</p>
+              <p className="chapter">{Math.floor(Math.random() * 10)}</p>
             </div>
```

Comment:
- [ ] Great job so far with the styling
**Your design**
![image](https://github.com/iamsjunaid/Bookstore/assets/62849872/6d94b1fd-2112-4cef-a5f6-6cbe5dd4fe41)
- Looking at the screenshot above, your vertical bar is double but in the zeplin design, the bar is single.
- Secondly, the chapter number should be `CHAPTER 8` and not just `8`
- The spacings between your elements seem to be far bigger. Try to adjust that also.
**Zeplin design**
![image](https://github.com/iamsjunaid/Bookstore/assets/62849872/bbd8fc59-2b87-4e82-b5df-56e24e60a290)

Reply:
Thanks for the feedback @t-yanick😊. I have made all the requested changes in the project.